src/shared/directives/conditional-show/services/condition-parser.service.spec.ts

```diff
@@ -0,0 +1,744 @@
+import { ConditionParser } from './condition-parser.service';
+
+describe('ConditionParser', () => {
+
+    describe('parse', () => {
```

Comment:
Indenting should all be 2 spaces rather than 4.

Reply:
Changed it to 2